tests/base.py

```diff
@@ -43,6 +43,7 @@ def setUp(self):
         app = self.create_app()
         self.app_test = app.test_client()
         with app.app_context():
+            print(Base.metadata.create_all(bind=engine))
```

Comment:
Hey @C3real-kill3r , please remember to remove this print statement If It's not required.

Reply:
Thanks @nakatuddesuzan noted